dev/com.ibm.ws.security.kerberos.auth/src/com/ibm/ws/security/kerberos/auth/Krb5LoginModuleWrapper.java

```diff
@@ -36,7 +36,7 @@
     public static final String COM_SUN_SECURITY_JGSS_KRB5_ACCEPT = "com.sun.security.jgss.krb5.accept";
 
     @FFDCIgnore(Throwable.class)
-    private static boolean isIBMLoginModuleAvailable() {
+    public static boolean isIBMLoginModuleAvailable() {
```

Comment:
This method should make use of the new JavaInfo.isAvailable method from #18011

Reply:
Suggested change is here: https://github.com/OpenLiberty/open-liberty/issues/18054